onyo/commands/cat.py

```diff
@@ -1,3 +1,4 @@
+import argparse
```

Comment:
Instead of importing `argparse`, consider the following (if `argparse` is only used for type annotations):

```
from __future__ import annotations
from typing import TYPE_CHECKING

if TYPE_CHECKING:
    import argparse

def cat(args: argparse.Namespace, opdir: str) -> None:
```

This only loads argparse when type checking.